create-resources-at-scale/create-resources-at-scale.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Passed validation in Azure Cloud Shell Bash environment on 4/25/2022
+
+# Variable block
+# These variables have placeholder values that are replaced with values from the csv input file.
+lineCounter="0"

```

Comment:
Add an **subscriptionName** parameter as every script needs this at the top -- there is no default subscription capability right now in azCLI.